src/main/java/ua/goit/timonov/enterprise/module_6_2/view/menus/DishesConsoleMenu.java

```diff
@@ -0,0 +1,117 @@
+package ua.goit.timonov.enterprise.module_6_2.view.menus;
+
+import ua.goit.timonov.enterprise.module_6_2.controllers.DishController;
+import ua.goit.timonov.enterprise.module_6_2.model.Dish;
+import ua.goit.timonov.enterprise.module_6_2.view.console.ConsoleIO;
+
+import java.util.List;
+
+import static ua.goit.timonov.enterprise.module_6_2.view.console.ConsolePrinter.printLine;
+
+/**
+ * Console menu for tasks with dishes that can be cooked in restaurant
+ */
+public class DishesConsoleMenu extends ConsoleMenu {
+
+    public static final String DISHES = "dishes";
+    public static final String DISH = "dish";
+    public static final String NAME = "name";
+    public static final String NO_SUCCESS = "UNSUCCESSFUL! There's no dish with such ";
+
+    /* controller for tasks with dishes */
+    private DishController dishController;
+
+    public void setDishController(DishController dishController) {
+        this.dishController = dishController;
+    }
+
+    /**
+     * configures menu's items:
+     * - get from DB list of all dishes
+     * - add new dish to DB
+     * - search dish by ID
+     * - search dish by name
+     * - delete dish from DB by ID
+     * - search dish from DB by name
+     */
+    public DishesConsoleMenu() {
+
+        addItem(new ConsoleMenuItem("Get all dishes") {
+            @Override
+            public void run() {
+                try {
+                    List<Dish> dishes = dishController.getAll();
+                    ConsoleIO.outputDishes(dishes);
+                } catch (RuntimeException e) {
```

Comment:
Such try-catches pollutes code. Handle exceptions in some abstract menu item. Refactoring needed